tools/sklearn/train_test_eval.xml

```diff
@@ -1,11 +1,11 @@
-<tool id="sklearn_train_test_eval" name="Train, Test and Evaluation" version="@VERSION@">
+<tool id="sklearn_train_test_eval" name="Train, Test and Evaluation" version="@VERSION@" profile="20.05">
```

Comment:
Do you think this tool is still needed? I thought it was deprecated a while ago.

Reply:
I don't have a feeling for this. @anuprulez ?